stockApp/stockApp/UseCase/GetAssetUseCase.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+enum UseCaseError: Error, Equatable {
+    case error
+}
+
+class GetAssetUseCase: GetAssetUseCaseProtocol {
+    
+    var manager: AssetManager
+    
+    init(manager: AssetManager) {
```

Comment:
```suggestion
    init(manager: AssetManagerProtocol) {
```

you should use a protocol instead of a concreat class